docs/content/s3.md

```diff
@@ -3029,7 +3029,7 @@ to normal storage.
 
 Usage Examples:
 
-    rclone backend restore s3:bucket/path/to/object [-o priority=PRIORITY] [-o lifetime=DAYS]
+    rclone backend restore s3:bucket/path/to --include object [-o priority=PRIORITY] [-o lifetime=DAYS]
```

Comment:
I think this should be `--include /object` otherwise you'll restore any objects under the file heirachy called `object`.

I'd also leave the `/` on the end of `s3:bucket/path/to/` which signals to rclone that this is definitely a directory.

Reply:
I think that's done now, sorry, new to using github, but this should be updated now?

Reply:
> I think that's done now, sorry, I'm new to using github, but there should be a new "tweaked path slashes" commit with these changes now?

Yes there are two - now you should squash them into one - edit commit message - and force push. It will make it one again.

Reply:
git cmd can be challenging - myself I use GitKraken (free for public repos) and recommend it to everybody:)

Reply:
yeah, still trying to crack it. Almost there...

Reply:
PR should contain one commit only - so squashing and forced pushing is daily bread for PR authors:)

Reply:
I've squashed them, but when trying to `git push`  them back it kept rejecting them `(non-fast-forward)`
I hope that it won't cause any issue that I "forced that push": `git push -f`
It seems to be okay now.

Reply:
yeap - they have to forced push - and all is now how it should:)

Reply:
Its easy for me to squash commits into one when merging - there is a nice button on GitHub which does that. So two commits would have been fine.

However one is even better :-)